### PR TITLE
feature/1021_add_cygnus_common_docker

### PR DESCRIPTION
**Pull request by pcoello25**
* Fix issue #1021 
* No tests are required
* Assignee: @frbattid

### CODE REVIEW

File: docker/cygnus-common/README.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+REPOSITORY          TAG                 IMAGE ID            CREATED             VIRTUAL SIZE
+cygnus-common       latest              0d2e537ac922        41 minutes ago      673.8 MB
+centos              6                   61bf77ab8841        6 weeks ago         228.9 MB                          
+```
+
+[Top](#top)
+
+##<a name="section3"></a>Using the image
+###<a name="section3.1"></a>As it is
+The cygnus-common image (either built from the scratch, either downloaded from hub.docker.com) allows running a Cygnus agent in charge of log messages at INFO level, known as [logger-sink](https://flume.apache.org/FlumeUserGuide.html#logger-sink).
```
Comment (frbattid) [inline docker/cygnus-common/README.md]: of logging

Comment (frbattid) [inline docker/cygnus-common/README.md]: know as --> ". This is because the default agent configuration runs a logger sink".

Comment (pcoello25) [inline docker/cygnus-common/README.md]: Fixed in 6fea2ff

**Comment by frbattid** [comment]
LGTM

**Pull request merged**